src/commands/pickFuncProcess.ts

```diff
@@ -32,32 +33,37 @@ export async function startFuncProcessFromApi(
         error: ''
     };
 
-    const uriFile: vscode.Uri = vscode.Uri.file(buildPath)
-
-    const azFuncTaskDefinition: AzureFunctionTaskDefinition = {
-        // VS Code will only run a single instance of a task `type`,
-        // the path will be used here to make each project be unique.
-        type: `func ${uriFile.fsPath}`,
-        functionsApp: uriFile.fsPath
-    }
-
     let funcHostStartCmd: string = 'func host start';
     if (args) {
         funcHostStartCmd += ` ${args.join(' ')}`;
     }
 
     await callWithTelemetryAndErrorHandling('azureFunctions.api.startFuncProcess', async (context: IActionContext) => {
         try {
-            await waitForPrevFuncTaskToStop(azFuncTaskDefinition.functionsApp);
-            const funcTask = new vscode.Task(azFuncTaskDefinition,
-                vscode.TaskScope.Global,
-                hostStartTaskName, 'func',
+            let workspaceFolder: vscode.WorkspaceFolder | undefined = buildPathToWorkspaceFolderMap.get(buildPath);
+
+            if (workspaceFolder === undefined) {
+                workspaceFolder = {
+                    uri: vscode.Uri.parse(buildPath),
+                    name: buildPath,
+                    index: -1
```

Comment:
I'm also not sure if the consequences of using index: -1 if VS Code actually looks at this 😓